server/src/main/java/org/spine3/server/command/CommandBus.java

```diff
@@ -326,18 +277,36 @@ public void run() {
         return result;
     }
 
-    /** Obtains the instance of the {@link CommandStatusService} associated with this command bus. */
+    /**
+     * Obtains the instance of the {@link CommandStatusService} associated with this command bus.
+     */
     public CommandStatusService getCommandStatusService() {
         return commandStatusService;
     }
 
+    /* package */ boolean isMultitenant() {
```

Comment:
We don't have `setMultitenant()` in the `Builder`.  We have the internal method in `CommandBus` for setting (because it's invoked by `BoundedContext`). And the query method is package access, because we call it from helper classes (in the same package).